### PR TITLE
An more tests

### PR DESCRIPTION
**Pull request by AndreaNeumayr**
turn on and off collision handling via action commands

### CODE REVIEW

File: src/Composition/dynamics.jl [hunk 1]
```diff
@@ -139,11 +139,16 @@ function initSegment_Model3D!(partiallyInstantiatedModel::Modia.InstantiatedMode
...
-                @error("Andrea: print warning für gripping features")
-                # printWarnGrip(robotOrDepot, movableObj, waitingPeriod)
+            if isdefined(scene, :gripPair)
+                if Modia3D.checkGrippingFeatures(scene, scene.gripPair)
+                    Modia3D.changeParentOfMovableUnit!(scene, world, scene.gripPair)
+                    if isdefined(scene.gripPair, :enableContactDetection)
+                        scene.collide = scene.gripPair.enableContactDetection
+                    end
+                else
+                    @error("Andrea: print warning für gripping features")
```
Comment (GerhardHippmann) [inline src/Composition/dynamics.jl:149]: Do you want to keep "Andrea" in the error message?

**Comment by GerhardHippmann** [comment]
Run tests fail because of https://github.com/ModiaSim/Modia.jl/issues/169. I think we need a new Modia minor release to fix this.

**Comment by MartinOtter** [comment]
> Run tests fail because of [ModiaSim/Modia.jl#169](https://github.com/ModiaSim/Modia.jl/issues/169). I think we need a new Modia minor release to fix this.

Released Modia 0.12.1

**Comment by GerhardHippmann** [comment]
Why did you disable plots?

**Pull request merged**